### ISSUE TITLE
new releases needed

### ISSUE THREAD

**Issue by DanielRuf** [sourcegraph/checkup#113]
https://github.com/sourcegraph/checkup/releases

It seems there was no new tag or release since 2017 and the docs are not very helpful to get a new one.

**Comment by titpetric** [comment]
I think we won't add new releases to GH any time soon. Can you try the following and check if it works for you:

~~~bash
curl -sf https://gobinaries.com/sourcegraph/checkup | sh
~~~

I also predict that it won't be usable soon, as it wouldn't pack the needed assets into the binary. Packing those into the binary will be done some time in the near future.

**Comment by DanielRuf** [comment]
Tried it and got a 500 error:

```
➜  checkup-test curl -sf https://gobinaries.com/sourcegraph/checkup | sh

  ==> Downloading github.com/sourcegraph/checkup/v2/@master
  ==> Resolved version master to v2.0.0
  ==> Downloading binary for darwin amd64

  Error downloading, got 500 response from server
```

**Comment by titpetric** [comment]
Okay, thank you for trying. I can’t promise an official release at this
time, but I hope I can get to it sooner  rather than later. It’s definitely
in the pipeline.
